ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace Homework
+{
+    using System;
+    using System.IO;
+    using System.Collections.Generic;
+    using System.Text;
+    using System.Formats;
+    using System.Text.RegularExpressions;
+
+    public static class MyExtensions
+    {
+        public static bool IsWeekend(this DateTime date)
+        {
+            if (date.DayOfWeek == DayOfWeek.Sunday || date.DayOfWeek == DayOfWeek.Saturday) return true;
+            else return false;
+        }
+
+        public static bool IsIsWorkday(this DateTime date)
+        {
+            return !IsWeekend(date);
+        }
+
+        public static int[,] ChunkBy(this int[] array, int chunk)
+        {
+            int c = chunk;
+            int count = array.Length / chunk;
+            int[,] splited = new int[count,chunk];
```

Comment:
multidimensional array is not the same as array of array, here is second option preferable